### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by iaheck**
landing changes if user is logged in:
image

or not:
image

### CODE REVIEW

File: components/home/welcomeBack.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <h1 className={styles.title}>Welcome back to Paraffin,</h1>
+      <h1 className={styles.title}>{userName}</h1>
+      <p className={styles.description}>
+        Your path towards your next career as a software developer awaits
+      </p>
+
+      <div className={styles.grid}>
+        <Link href="/curriculums/1" className={styles.card}>
+          <h2>Resume &rarr;</h2>
+          <p>Ready to continue? Resume your learning journey here.</p>
```
Comment (rtichauerv) [inline components/home/welcomeBack.js:16]: All these elements could use the Typography component, just to have in mind when you check the styling stuff.

**Review by rtichauerv — CHANGES_REQUESTED** [review]
Very good in general, some comments that you can ignore if you want, that are more of an opinion than anything else, just 1 that really matters about the error retrieved for passing 2 components to the Link Component in the welcomeBack component.

Also, components files should use Pascal Case, all the welcomeStuff and the homeContent are not complying this

**Comment by rtichauerv** [comment]
Now I have this error, for calling useContext in the use-sign-out hook
image

File: components/home/welcomeBack.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <main className={styles.main}>
+      <h1 className={styles.title}>Welcome back to Paraffin,</h1>
+      <h1 className={styles.title}>{userName}</h1>
+      <p className={styles.description}>
+        Your path towards your next career as a software developer awaits
+      </p>
+
+      <div className={styles.grid}>
+        <Link href="/curriculums/1" className={styles.card}>
```
[the remark this answers is not in the document]
Comment (iaheck) [reply components/home/welcomeBack.js]: done! with material components instead

File: components/home/welcomeBack.js (hunk 1 above)
Comment (iaheck) [reply components/home/welcomeBack.js]: donee

File: components/home/welcomeUnknown.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          <p>
+            You couldn&apos;t have found a better place to start your career.
+          </p>
+        </a>
+
+        <a href="/users/sign_in" className={styles.card}>
+          <h2>Log in &rarr;</h2>
+          <p>Already have an account? Continue your learning journey here.</p>
+        </a>
+      </div>
```
[the remark this answers is not in the document]
Comment (iaheck) [reply components/home/welcomeUnknown.js]: done as well

File: pages/curriculums/[curriculumId].js [hunk 4]
```diff
@@ -10,5 +10,10 @@ export default function LearningUnits() {
   }
 
   const { curriculumId } = query;
-  return <LearningUnitsOfCurriculum curriculumId={curriculumId} />;
+
+  return (
+    <>
+      <LearningUnitsOfCurriculum curriculumId={curriculumId} />;
+    </>
+  );
```
[the remark this answers is not in the document]
Comment (iaheck) [reply pages/curriculums/[curriculumId].js:18]: you're right, that was unintended

**Review by rtichauerv — APPROVED** [review]
Very good changes!

**Pull request merged**